### ISSUE TITLE
Since yesterday, failure to send responses to Skype users

### ISSUE THREAD

**Issue by pfeifel** [Microsoft/BotBuilder-V3#97]
## Version
Using REST API to https://smba.trafficmanager.net/apis/v3/
Using C# HTTP client

## Describe the bug
Since yesterday, sending any message to this endpoint results an exception: "Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host."
This exact configuration worked flawlessly until yesterday.

To clarify, we are getting incoming messages. The problem is when posting the responses.

**Comment by n0sleep** [comment]
I'm not sure if its the same root error, but I also got the same exception "Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host" since yesterday.

I'm using the directline c# client (nuget package version 3.0.2) and the error happens when I simply call:
_directLineClient.Tokens.GenerateTokenForNewConversation()

**Comment by CoHealer** [comment]
@n0sleep, can you please send the bot handle so we can look it up in the service logs?

**Comment by pfeifel** [comment]
I found the issue and fixed my code to avoid the problem, as follows:

I am using HttpWebRequest with .NET framework 4.6.
The exception I got was as described above, but digging into the call stack, I saw that the exception was generated from the internal .NET function "System.Net.TlsStream.ProcessAuthentication". This gave me the hint to investigate the security protocol.

Under .NET 4.6, the ServicePointManager.SecurityProtocol property defaults to "SecurityProtocolType.Ssl3 | SecurityProtocolType.Tls". This used to work when connecting to the above Skype endpoint until recently, and then it stopped working.

I changed my client code, setting ServicePointManager.SecurityProtocol before creating the request to "SecurityProtocolType.Ssl3 | SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12". This fixed the problem.

Apparently, the minimum security protocol requirements on the Skype servers were changed.
